src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/GetTotalAndSumContentLengthIndexedAsyncV57.sql

```diff
@@ -0,0 +1,19 @@
+/***************************************************************************************/
+-- STORED PROCEDURE
+--     GetTotalAndSumContentLengthIndexedAsyncV57
+--
+-- FIRST SCHEMA VERSION
+--     57
+--
+-- DESCRIPTION
+--     Retrieves total sum of content length across all FileProperty rows
+--
+/***************************************************************************************/
+CREATE OR ALTER PROCEDURE dbo.GetTotalAndSumContentLengthIndexedAsyncV57
+    AS
+BEGIN
+    SET NOCOUNT     ON
+    SET XACT_ABORT  ON
+
+SELECT TotalIndexedFilesCount=COUNT(*), TotalIndexedBytes=SUM(ContentLength) FROM dbo.FileProperty
```

Comment:
I remember from your previous PR you set the contentLEgnth to -1. do you want to discard those here?